### PR TITLE
[MM-25659] document permanent delete of channel

### PR DESCRIPTION
**Pull request by ashishbhate**
#### Summary

- Document permanent delete of channel

#### Ticket Link

- https://mattermost.atlassian.net/browse/MM-25659

### CODE REVIEW

File: v4/source/channels.yaml
Comment (isacikgoz) [inline v4/source/channels.yaml:599]:
```suggestion
        As of server version 5.28, optionally use the `permanent=true` query parameter to permanently delete the channel for compliance reasons. To use this feature `ServiceSettings.EnableAPIChannelDeletion` must be set to `true` in the server's configuration.
```

**Review by mgdelacroix — APPROVED** [review]

**Comment by hanzei** [comment]
@ashishbhate I think this is fine to merge as it's clearly document when the changes will come into effect.

**Comment by justinegeffen** [comment]
This addresses: https://github.com/mattermost/mattermost-server/pull/15202.

**Comment by ashishbhate** [comment]
@justinegeffen is this PR (and #562 ) good to merge?

**Comment by justinegeffen** [comment]
@ashishbhate, I think @hanzei is the best person to ask as I don't have write access to this repo. Sorry! :(

**Pull request merged**
